src/ceph-volume/ceph_volume/devices/raw/list.py

```diff
@@ -18,6 +19,27 @@ def direct_report(devices):
     _list = List([])
     return _list.generate(devices)
 
+def _get_bluestore_info(dev):
+    out, err, ret = process.call([
+        'ceph-bluestore-tool', 'show-label',
+        '--dev', dev], verbose_on_failure=False)
+    if ret:
+        logger.debug('failed to get BlueStore info on device %s' % dev)
+        return None
```

Comment:
hiding the error? is it on purpose?

Reply:
This is just copied from what was pre-existing. I think this is still okay since it will merely skip devices where the `ceph-bluestore-tool` command fails, and this is expected to fail when disks don't have a bluestore label.

Reply:
I think this *is* an error when we have already checked if the disk has the bluestore header though. So I will add a check in the caller when appropriate.

Reply:
https://github.com/ceph/ceph/pull/42469/files#r676766574